code/src/components/Style.js

```diff
@@ -0,0 +1,134 @@
+import styled from "styled-components/macro"
```

Comment:
I like your approach of using styled components in a separate component! It had not crossed my mind we can write it like this :) The only con to this, I think, is that it's not as easy to follow the code BUT when it comes to you guys I think you really nailed it since you named your consts in a very good way. I think this is very crucial when styling the code in a separate file (as would be in a "normal" css file). So, thumbs up! :)